Roles/Neutral/FFF.cs

```diff
@@ -0,0 +1,138 @@
+{
+    public static class FFF
+    {
+        private static readonly int Id = 11300;
+        public static List<byte> playerIdList = new();
+        public static bool IsEnable = false;
+
+        public static OptionItem CanVent;
+        public static OptionItem ChooseConverted;
+        public static OptionItem MisFireKillTarget;
+
+        public static OptionItem CanKillLovers;
+        public static OptionItem CanKillMadmate;
+        public static OptionItem CanKillCharmed;
+        public static OptionItem CanKillAdmired;
+        public static OptionItem CanKillSidekicks;
+        public static OptionItem CanKillEgoists;
+        public static OptionItem CanKillInfected;
+        public static OptionItem CanKillContagious;
+
+        public static List<byte> winnerFFFList = new();
+        public static void SetupCustomOption()
+        {
+            SetupRoleOptions(Id, TabGroup.NeutralRoles, CustomRoles.FFF, zeroOne: false);
+            MisFireKillTarget = BooleanOptionItem.Create(Id + 11, "FFFMisFireKillTarget", true, TabGroup.NeutralRoles, false).SetParent(CustomRoleSpawnChances[CustomRoles.FFF]);
+            ChooseConverted = BooleanOptionItem.Create(Id + 12, "FFFChooseConverted", true, TabGroup.NeutralRoles, false).SetParent(CustomRoleSpawnChances[CustomRoles.FFF]);
+            CanKillMadmate = BooleanOptionItem.Create(Id + 13, "FFFCanKillMadmate", true, TabGroup.NeutralRoles, false).SetParent(ChooseConverted);
+            CanKillCharmed = BooleanOptionItem.Create(Id + 14, "FFFCanKillCharmed", true, TabGroup.NeutralRoles, false).SetParent(ChooseConverted);
+            CanKillLovers = BooleanOptionItem.Create(Id + 15, "FFFCanKillLovers", true, TabGroup.NeutralRoles, false).SetParent(ChooseConverted);
+            CanKillSidekicks = BooleanOptionItem.Create(Id + 16, "FFFCanKillSidekick", true, TabGroup.NeutralRoles, false).SetParent(ChooseConverted);
+            CanKillEgoists = BooleanOptionItem.Create(Id + 17, "FFFCanKillEgoist", true, TabGroup.NeutralRoles, false).SetParent(ChooseConverted);
+            CanKillInfected = BooleanOptionItem.Create(Id + 18, "FFFCanKillInfected", true, TabGroup.NeutralRoles, false).SetParent(ChooseConverted);
+            CanKillContagious = BooleanOptionItem.Create(Id + 19, "FFFCanKillContagious", true, TabGroup.NeutralRoles, false).SetParent(ChooseConverted);
+            CanKillAdmired = BooleanOptionItem.Create(Id + 20, "FFFCanKillAdmired", true, TabGroup.NeutralRoles, false).SetParent(ChooseConverted);
+        }
+
+        public static void Init()
+        {
+            playerIdList = new();
+            IsEnable = false;
+            winnerFFFList = new();
+        }
+
+        public static void Add(byte playerId)
+        {
+            playerIdList.Add(playerId);
+            IsEnable = true;
+
+            if (!AmongUsClient.Instance.AmHost) return;
+            if (!Main.ResetCamPlayerList.Contains(playerId))
+                Main.ResetCamPlayerList.Add(playerId);
```

Comment:
You can remove this line as Hater doesn't keep the game going, and therefore it doesn't need to trigger anti blackout

Reply:
I can PR these changes in like half an hour